android/java/res/layout/brave_shields_secondary_layout.xml

```diff
@@ -58,4 +58,15 @@
 		android:id="@+id/brave_shields_fingerprinting_switch_id"
 		layout="@layout/brave_shields_switcher"/>
 
+    <TextView
+        android:id="@+id/brave_shields_block_element_text"
+		android:layout_width="wrap_content"
+		android:layout_height="48dp"
+		android:paddingStart="16dp"
```

Comment:
as `paddingStart` and `paddingEnd` are the same, so it is possible to use a single `android:paddingHorizontal`